### ISSUE TITLE
[Feature Request] Firearm Execution

### ISSUE THREAD

**Issue by stormcloud23** [d3athrow/vgstation13#24860]
<featurerequest>
I'd like to hear opinions on adding a feature to stuff like the NT glock or other low powered weapons (or maybe all firearms, the pistols are just the ones that would have the most impact) that allows for execution without having to magdump 8 rounds into the skull. The way I think about it, you would only be able to do this to someone who was cuffed and being pulled by you (maybe buckle cuffed), and it would have a progress bar, similar to what happens when you target the mouth and click on yourself, where it says "X puts the gun in his mouth and prepares to pull the trigger!"

Obviously it makes sense that pistols as a whole shouldn't do as much damage as laser rifles, but when you've got a confirmed cultist cuffed and the authorization for lethal force, it makes sense to just be able to line up a kill shot between the eyes and get it done, rather than having to stand there shooting them multiple times or stabbing/clubbing them to death to try and save ammo. Thoughts?

**Comment by boy2mantwicethefam** [comment]
I think there's already a rudimentary form that just multiplies damage by 1.5x if you fire it point blank at someone on harm intent and clicking them

**Comment by stormcloud23** [comment]
I know harm intent allows you to shoot point blank, rather than just pistol whipping. Didn't know about the damage increase if its there, which yeah is kinda what I'm talking about, but 1.5x the damage from a .380 still isn't really representative of a shot to the head on someone who cant fight back. It makes sense that shooting someone from point blank does more damage even in a regular fight. I just think there should be an option to use it as an execution tool for cuffed prisoners, both for RP reasons and to save on ammo. It just kinda seems a bit silly to me that, if you wanna do it with a pistol, you've gotta shoot them like 5 times in the head, meanwhile they might still be yelling or whatever else. It just kinda takes you out of it. Like "Why can't my character hit the brain on a guy who is not moving and who is literally standing in arms reach?" Maybe there's balance concerns, but I still think there could be a way to make it work.

**Comment by utilitas-** [comment]
\>NT glock
You mean the NT Piss Clock?
You literally do more damage by throwing the magazine than by shooting them with lethals with the NT BB Glock.

**Comment by stormcloud23** [comment]
>You mean the NT Piss Clock?

Yeah. I mean, everything you just said is kinda part of the issue too.

**Comment by Irradiation** [comment]
It's a nice idea and I agree that it makes sense from a realism/RP perspective,, but there's an argument somewhere about """balance""" in terms of being able to execute a guy with a single shot from a .380, as long as you have a timed delay. Keep in mind that Magstomping does 50 brute, requires the victim to be on the floor and there's a 2-3 seconds per magstomp.

Also selective realism is weird and wonky.
